docs/examples/CustomFields/RangeField.js

```diff
@@ -37,6 +53,9 @@ export default function ExamplesRangeField() {
 
   return (
     <AutoForm
+      model={{
+        range: { start: today, stop: tommorow }
+      }}
```

Comment:
It'd be better to create the model only once, to keep the reference. Also, using a fixed pair of dates (not _today_, _tomorrow_) will simplify it a bit.